File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/AudioFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.std.files.Audio;

public class AudioFilterValidator extends FilterValidator<Audio> {

    public AudioFilterValidator(CustomValidator<Audio> validator) {
        super(Audio.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/AudioFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.telegram.bot.api.model.objects.std.files.Audio
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class AudioFilterValidator(validator: Validator<Audio>) : FilterValidator<Audio>(Audio::class, validator)

